maven/bom/pom.xml

```diff
@@ -76,8 +76,13 @@
             </dependency>
             <dependency>
                 <groupId>org.daisy.pipeline.modules.braille</groupId>
-                <artifactId>dtbook-to-pef</artifactId>
-                <version>3.3.1-SNAPSHOT</version>
+                <artifactId>xml-to-pef</artifactId>
```

Comment:
Oh, I didn't realize. I thought I'd put it so that the things below depends on the stuff above, but alphabetic would be fine :)